tests/doc/run_doc_python_server_api_examples.py

```diff
@@ -2,6 +2,7 @@
 from time import sleep
```

Comment:
also, this is to emulate the docs, will not run as a test
@sbellem maybe we should convert this to function as the driver docs and encapsulate the behavior in tests instead of this runfile